<ui version="4.0" >
 <class>convertDialog</class>
 <widget class="QDialog" name="convertDialog" >
  <property name="windowModality" >
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>126</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>执法督查科图片批量处理工具</string>
  </property>
  <property name="windowIcon" >
   <iconset>
    <normaloff>logo.jpg</normaloff>logo.jpg</iconset>
  </property>
  <property name="modal" >
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item row="0" column="0" >
    <layout class="QGridLayout" name="gridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="dirLabel" >
       <property name="text" >
        <string>图片源目录：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="dirLineEdit" />
     </item>
     <item row="0" column="2" >
      <widget class="QPushButton" name="dirButton" >
       <property name="text" >
        <string>浏览</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="rateLabel" >
       <property name="text" >
        <string>转换比例：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="rateLineEdit" />
     </item>
     <item row="1" column="2" >
      <widget class="QPushButton" name="convertButton" >
       <property name="text" >
        <string>转换</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="helpLabel" >
     <property name="text" >
      <string>1.
2
3</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="zoomtc.qrc" />
 </resources>
 <connections/>
</ui>
